features/admin.actions.v1/components/action-config-form.tsx

```diff
@@ -0,0 +1,841 @@
+                    handleError(error, ActionsConstants.UPDATE);
+                })
+                .finally(() => {
+                    setIsSubmitting(false);
+                });
+        }
+    };
+
+    const renderFormFields = (): ReactElement => {
+
+        const renderAuthenticationSection = (): ReactElement => {
+
+            const renderAuthenticationSectionInfoBox = (): ReactElement => {
+
+                const resolveAuthTypeDisplayName = (): string => {
+                    switch (authenticationType) {
+                        case AuthenticationType.NONE:
+                            return t("console:manage.features.actions.fields.authentication.types.none.name");
+                        case AuthenticationType.BASIC:
+                            return t("console:manage.features.actions.fields.authentication.types.basic.name");
+                        case AuthenticationType.BEARER:
+                            return t("console:manage.features.actions.fields.authentication.types.bearer.name");
+                        case AuthenticationType.API_KEY:
+                            return t("console:manage.features.actions.fields.authentication.types.apiKey.name");
+                        default:
+                            return;
+                    }
+                };
+
+                return (
+                    <Alert severity="info">
+                        <AlertTitle
+                            className="alert-title">
+                            <Trans
+                                i18nKey={
+                                    authenticationType === AuthenticationType.NONE ?
+                                        t("console:manage.features.actions.fields.authentication" +
+                                        ".info.title.noneAuthType") :
+                                        t("console:manage.features.actions.fields.authentication" +
+                                            ".info.title.otherAuthType", { authType: resolveAuthTypeDisplayName() })
+                                }
+                                // values={ { authType: resolveAuthTypeDisplayName() } }
+                                components={ { strong: <strong/> } }
+                            />
+                        </AlertTitle>
+                        <Trans
+                            i18nKey={ t("console:manage.features.actions.fields.authentication.info.message") }
+                        >
+                                If you are changing the authentication, be aware that the authentication secrets of
+                                the external endpoint need to be updated.
+                        </Trans>
+                        <div>
+                            <Button
+                                onClick={ handleAuthenticationChange }
+                                variant="outlined"
+                                size="small"
+                                className={ "mt-2" }
```

Comment:
do not use these prefixes, use a separate sass file instead.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/c514505d74137c56840758c44975197ff519db64